config/default.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
You're going to want to not check this in.
You're going to want to change your passwords.